ourjapanlife/findadoc-server
# chore: update insurance enum

# What changed
- Creates the `acceptedInsurance` column in the `HealthcareProfessional` table.
- Adds insurance data to the seeds.
- Removes `UNKNOWN` from the `enum Insurance` in favor of leaving the field blank if it isn't known.

# Testing instructions
1. Delete the database
2. Run `docker compose up`
3. Run `yarn prisma generate`
4. Run `yarn prisma studio`
5. Confirm you can see the `acceptedInsuranceOptions` on the `HealthcareProfessional`

## Screenshot
![Screenshot 2023-01-22 at 9 54 36 PM](https://user-images.githubusercontent.com/31802656/213916857-1e6183fb-1d4a-4788-8f29-c99672fc9a22.png)
